codeChallenges/challenge-1/index.test.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Given a single input array, write a function that outputs an unique array of values
+ *
+ *
+ * What time complexity is your solution?
+ *
+ * Your solution shouldn't using any build in function.
+ *
+ * Credit: Write a unit tests.
+ */
+/**
+  * example usage:
+  * const arrOfNum = [1, 2, 2, 4, 5, 6, 6];
+  * console.log(getUniqueValues(arrOfNum)); // [1, 2, 4, 5, 6]
+  */
+ const arrOfNum = [1, 2, 2, 4, 5, 6, 6];
+ const getUniqueValues = (arrOfNum) => {
+   // TO DO
+   var uniqueNums = [];
+   for(let i = 0; i < arrOfNum.length; i++){
+    if(arrOfNum[i+1] != arrOfNum[i]){
```

Comment:
Here you are checking tow values throw itteration, if they are equaled or on not,
what about [1,2,5,2] ? here if the duplicated values are not follow each other you code will not cover this case.

Reply:
resolve at: [refrence](https://github.com/RaghadAbdulhadi1/Training-Raghad/pull/1/commits/d433e02deb8c28c50cbcd84db4fcd28d292eee02)